src/palladio/ResolveMapCache.cpp

```diff
@@ -36,23 +38,27 @@ bool isEmbedded(const std::filesystem::path& p) {
 	return startsWithAnyOf(p.string(), EMBEDDED_SCHEMAS);
 }
 
+#ifndef PLD_TEST_EXPORTS
 UT_StringHolder getFSReaderFilename(const FS_Reader& fsr) {
-#if HOUDINI_VERSION_MAJOR > 16
+#	if HOUDINI_VERSION_MAJOR > 16
 	return fsr.getFilename();
-#else
+#	else
 	UT_String fsrFileName;
 	fsr.getFilename(fsrFileName);
 	return UT_StringHolder(fsrFileName);
-#endif
+#	endif
```

Comment:
The if HOUDINI_VERSION_MAJOR > 16 is redundant now as we only support H18.5+

Reply:
done